src/commonTest/resources/dev/rhovas/interpreter/programs/rosettacode/Fibonacci.rho

```diff
@@ -0,0 +1,26 @@
+func pfib(n: Integer): Integer {
+    require n >= 0;
+
+    var lastFib = -1;
+    var fib = 1;
+    range(0, n, :incl).for {
+        var sum = fib + lastFib;
```

Comment:
Use `val`, since it's not modified. I'd probably call this `next` as well; preferring a semantic description of usage rather than how the value is calculated.

Reply:
Gotcha!